app/models/concerns/base_state_machine.rb

```diff
@@ -83,6 +83,9 @@ class BaseStateMachine < ApplicationRecord  # rubocop:disable Metrics/ClassLengt
                   after: proc { |reason|
                     raise 'Invalid ccms_reason' unless reason.in?(VALID_CCMS_REASONS)
 
+                    # logging to find out where unknonw ccms reasons are coming from
+                    Raven.capture_message "UNKNOWN CCMS REASON\n#{caller}" if reason.to_s == 'unknown'
```

Comment:
couldn't you just do `reason == :unknown` to save having to create strings?

Reply:
Well, I don't know if the reason is being passed in as a symbol or a string, so this makes sure it captures both possibilities

Reply:
`reason.to_sym == :unknown` then?